data/json/items/tools.json

```diff
@@ -8307,6 +8307,35 @@
     "use_action": "BLOOD_DRAW",
     "flags": [ "SPEAR" ]
   },
+  {
+    "id": "hand_crank_charger",
+    "type": "TOOL",
+    "category": "tools",
+    "name": "hand-crank charger",
+    "description": "This is a hand-powered battery charger. It has an adjustable receptacle designed to accept a wide variety of rechargable battery cells.",
```

Comment:
```suggestion
    "description": "This is a hand-powered battery charger.  It has an adjustable receptacle designed to accept a wide variety of rechargeable battery cells.",
```

Reply:
I have SO much trouble with that word, my brain insists there must only be one `e` in it. Thanks for spotting that.

Reply:
You're welcome. That's mostly not me, but spellchecker in my browser.